NuKeeper/Engine/RepositoryUpdater.cs

```diff
@@ -98,7 +100,7 @@ private void GitCloneToTempDir()
                 Console.WriteLine("Commiting");
 
                 var commitMessage = CommitReport.MakeCommitMessage(updateSet);
-                _git.Commit(commitMessage);
+                _git.Commit($":{CommitEmoji}: {commitMessage}");
```

Comment:
The reason I didn't was cause you can't use emojis for PR titles, so I'm being lazy.
How about `MakeCommitMessage` returns a type with a method that prints it, and it takes an enum to distinguish between commit message and PR title?